benchmarks/experimental/offload.py

```diff
@@ -403,7 +405,7 @@ def run_benchmark(args):
 
 parser = argparse.ArgumentParser(description="benchmark")
 parser.add_argument(
-    "--dry_run", default=True, action="store_true", help="Run a sample training run without regression testing."
+    "--dry_run", default=False, action="store_true", help="Run a sample training run without regression testing."
```

Comment:
the default change is on purpose I presume, not a typo ?

Reply:
Yes, thats right. We don't want dry runs anymore but instead want to validate the numbers. thanks for checking!